### DISCUSSION TITLE
Question about training smoke on waymo

### DISCUSSION THREAD

**Discussion by JiSuanJiDaWang** [open-mmlab/mmdetection3d#2053] [Q&A] [answered]
Hi, I am trying to train the smoke model on waymo dataset based on the mmd3d version 1.1. The smoke model has its own method to convert the predefined angle to the alpha on kitti dataset. I found that the mmd3d has unified coordinate system. Do I need to define the method to map the angle output by smoke to the mmd3d coordinate? Thanks

**Comment by ZCMax** [comment, marked as answer]
The output of smoke model should be the yaw, not alpha. It should be easy to adapt smoke model to waymo dataset

**Reply by JiSuanJiDaWang** [reply]
Thanks so much! Do you mean I do not need to do any modification on the decoder of yaw since the mmd3d unified the coordinate systems?

**Reply by ZCMax** [reply]
yes
